tests/conftest.py

```diff
@@ -34,3 +34,17 @@ def phenopacket_store(
         yield PhenopacketStore.from_release_zip(
             zip_file=zip_file,
         )
+
```

Comment:
Can we please move the `PhenopacketStore` fixtures (including the one on line #30) closer to their use? Now it seems to me that then it will be easier to track their use.